### PR TITLE
Workflow for automated acceptance test cases

### PR DESCRIPTION
**Pull request by siddharth-kulshrestha**
1. Moved all the test case dependencies to a single file "test_config.json".
2. Added a workflow (acceptance-test.yml) to run the acceptance test cases on a self-hosted runner.
3. ok-to-test.yml uses slash command dispatch to dispatch execution of acceptance tests once `/ok-to-test` command is passed as an issue comment on any PR.

### CODE REVIEW

File: .github/workflows/acceptance-test.yml [hunk 1]
```diff
@@ -0,0 +1,83 @@
...
+      - name: Check out repository code
+        uses: actions/checkout@v2
+        with:
+          ref: 'refs/pull/${{ github.event.client_payload.pull_request.number }}/merge'
+      - name: Set up Go
+        uses: actions/setup-go@v2
+        with:
+          go-version: 1.17
+      - name: Acceptance test cases
+        run: CGO_ENABLED=0 NUTANIX_USERNAME="admin" NUTANIX_PASSWORD="Nutanix.123" NUTANIX_INSECURE=true NUTANIX_PORT=9440 NUTANIX_ENDPOINT="10.51.144.11" NUTANIX_PE_PASSWORD="Nutanix.123" NUTANIX_PE_USERNAME="admin" NUTANIX_STORAGE_CONTAINER="80766075-f2aa-4aee-a526-63d4a047d2ab" make testacc
```
Comment (shreevari) [inline .github/workflows/acceptance-test.yml]: Could we move these to Github action secrets like [here](https://github.com/nutanix/terraform-provider-nutanix/pull/325/files#diff-0c318bde7aec58c08f04a0656294734b3e1a75793b05a4443f9a635a3920157fR62)?

**Review by shreevari — COMMENTED** [review]
Maybe we could intentionally break the acceptance tests once and then fix them and run the acceptance tests again to see them in action.

File: .github/workflows/acceptance-test.yml (hunk 1 above)
Comment (siddharth-kulshrestha) [reply .github/workflows/acceptance-test.yml]: Yeah makes sense

**Review by venkatavivek-ntnx — APPROVED** [review]

**Review by shreevari — APPROVED** [review]

**Pull request merged**